src/main/java/org/folio/services/validator/registry/ValidatorRegistryServiceImpl.java

```diff
@@ -175,7 +176,7 @@ private Criteria constructCriteria(String jsonbField, String value) {
     Criteria criteria = new Criteria();
     criteria.addField(jsonbField);
     criteria.setOperation("=");
-    criteria.setValue(value);
+    criteria.setVal(value);
```

Comment:
The method constructCriteria should be removed.
We know for sure that id and ruleId are always the same.
Therefore PostgresClient.getById can be used instead.

Reply:
Can you also update the documentation in rule.json, rule.sample, ruleCollection.sample by adding the "id" property?

Reply:
Fixed